### ISSUE TITLE
IDISP001 should warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#55]
```cs
namespace RoslynSandbox
{
    using System;

    public class Foo
    {
        public Foo()
        {
            IDisposable disposable;
            ↓disposable = new Disposable();
        }
    }
}
```
